sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -98,7 +98,7 @@ implementation supports.
 This extension adds a new enumeration of the architectures that can be tested.
```

Comment:
This comment is really for the table above, which you did not change.  Please add a new row to that table for version 2 of this specification, noting that the Nvidia and AMD architectures were added in version 2.

Reply:
thanks @gmlueck. done.